en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md

```diff
@@ -86,21 +86,54 @@ Following are the artifact configurations that we can use to implement this scen
 
 ## Build and run
 
+[Configure the ActiveMQ broker]({{base_path}}/install-and-setup/setup/brokers/configure-with-activemq).
+
 Create the artifacts:
 
-1. [Set up WSO2 Integration Studio]({{base_path}}/develop/installing-wso2-integration-studio).
-2. [Create an integration project]({{base_path}}/develop/create-integration-project) with an <b>ESB Configs</b> module and an <b>Composite Exporter</b>.
-3. Create the [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service), [endpoints]({{base_path}}/develop/creating-artifacts/creating-endpoints), [message stores]({{base_path}}/develop/creating-artifacts/creating-a-message-store) and [message processors]({{base_path}}/develop/creating-artifacts/creating-a-message-processor) with the configurations given above.
-4. [Deploy the artifacts]({{base_path}}/develop/deploy-artifacts) in your Micro Integrator.
+1. {!includes/build-and-run.md!}
+2. Create the [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service), [endpoints]({{base_path}}/develop/creating-artifacts/creating-endpoints), [message stores]({{base_path}}/develop/creating-artifacts/creating-a-message-store) and [message processors]({{base_path}}/develop/creating-artifacts/creating-a-message-processor) with the configurations given above.
+3. [Deploy the artifacts]({{base_path}}/develop/deploy-artifacts) in your Micro Integrator.
 
-[Configure the ActiveMQ broker]({{base_path}}/install-and-setup/setup/brokers/configure-with-activemq).
+Set up the back-end service:
+
+1. Download the [back-end service](https://github.com/wso2-docs/WSO2_EI/blob/master/Back-End-Service/axis2Server.zip).
+2. Extract the downloaded zip file and create two additional copies of the extracted folder.
+3. Open the `axis2.xml` file located in the `axis2Server/conf/` directory within each extracted folder.
+4. Update the port numbers in the `axis2.xml` file for each folder to `9001`, `9002`, and `9003` respectively.
+3. Open a terminal, navigate to the `axis2Server/bin/` directory inside the extracted folders.
+4. Execute the following command to start the axis2server with the SimpleStockQuote back-end service:
+
+   === "On MacOS/Linux/CentOS"   
+   ```bash
+   sh axis2server.sh
+   ```
+   === "On Windows"          
+   ```bash
+   axis2server.bat
+   ```
 
-You can analyze the message sent by the Micro Integrator to the secure service using TCPMon.
+Send the following request to invoke the service:
 
-On successful execution of the placeorder request, you will see the following message on the back-end:
```

Comment:
Why is this content removed?

Reply:
This doc doesn't have a sample request. So, I updated those things. 

Here, we mentioned this [here](https://github.com/wso2/docs-mi/pull/388/files#diff-ee2fb1d2853c20da213130764044b4ad008548a26d83ed8408c705958fcc89ceR139)

I got a different log in the back-end server when I tested it. So, I removed it.

Reply:
what is the log you got in the back-end server?

Reply:
Mon Aug 12 13:19:27 IST 2024 samples.services.SimpleStockQuoteService :: Generating quote for : IBM